.clitest/functions/.test.teardown

```diff
@@ -0,0 +1,12 @@
+#!/bin/zsh
+
+# restore git
+(( $+functions[git] )) &&
+    unfunction git
+
+# restore fpath
+fpath=( $fpath_ )
+
+# remove tempdir
+[[ -d "$T_TEMPDIR" ]] &&
+    rm -rf -- "$T_TEMPDIR"
```

Comment:
The double quotes here are unnecessary. Zsh does not split on whitespace unless told to do so.